README.md

```diff
@@ -147,7 +147,9 @@ If you encounter issues during installation or usage, here are some common solut
    - If using embedchain or other tools: `pip install 'crewai[tools]'`
 
 2. **Failed building wheel for tiktoken**
-   - Ensure Rust compiler is installed (see installation steps above)
+   - For mac:
+     - Ensure Rust compiler is installed before installing tiktoken: `brew install rust && pip install tiktoken`
+     - Ensure that your python is version is not 3.13 (otherwise, you will get the error `error: the configured Python interpreter version (3.13) is newer than PyO3's maximum supported version (3.12)` when installing tiktoken
```

Comment:
I think this is a bit redundant since we’ve already mentioned the supported Python versions.